src/openforms/registrations/contrib/microsoft_graph/config.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+
+from openforms.template.validators import DjangoTemplateValidator
+from openforms.utils.mixins import JsonSchemaSerializerMixin
+
+DEFAULT_FOLDER_PATH = "/open-forms/"
+
+
+def is_absolute(value: str) -> None:
+    path = Path(value)
```

Comment:
using `PurePosixPath` is more appropriate here to enforce forward slashes - on Windows for example this would break since Path resolves to WindowsPath